packages/node_modules/@ciscospark/widget-recents/src/selector.js

```diff
@@ -134,6 +134,10 @@ const getRecentSpaces = createSelector(
           };
         }
 
+        if (space.isDecrypting) {
+          constructedSpace.name = 'Decrypting Space...';
```

Comment:
This helps IE11 not have to encrypting space name displayed since they don't support the blurry filter.